### DISCUSSION TITLE
Welcome to Floating UI Discussions!

### DISCUSSION THREAD

**Discussion by atomiks** [atomiks/floating-ui#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by atomiks** [comment]
/cc @XhmikosR you will likely be interested in this project for Bootstrap 6 :) 

https://floating-ui.com

**Comment by XhmikosR** [comment]
Hey, @atomiks. Thanks for the info!

The project does look good, indeed. That being said, we are still shorthanded and we are still trying to clean up things from the popper x2 switch. I'd be happy to switch to Floating UI for v6 assuming someone helps us out to make the switch.

And we should probably rethink our whole approach so that consumers of Bootstrap are not affected a lot by this kind of changes; a bridge or an adapter if you like.

Unfortunately, we don't have an ETA for v6, but happy to ping you later.
